keyboards/xelus/rs60/readme.md

```diff
@@ -0,0 +1,19 @@
+# RS60
+
+60% PCB with daughterboard in collaboration with Bisoromi and Mekibo
+
+* Keyboard Maintainer: [Xelus22](https://github.com/Xelus22)
+* Hardware Supported: RS60
+* Hardware Availability: Custom Keyboard Group Buys
+
+Make example for this keyboard (after setting up your build environment):
+
+    make xelus/rs60:default
+
+Reset your keyboard in 3 ways:
+
+* Software reset on Fn + Backspace
```

Comment:
This is not 100% clear, because you have both `MO(1)` and `MO(2)` in the default keymap, and apparently reset is on layer 2, while other action commonly associated with “Fn” are on layer 1. Although this may not really matter much, because the keymap can be customized anyway.